ground/src/main/java/com/google/android/ground/system/NetworkManager.kt

```diff
@@ -61,11 +61,13 @@ class NetworkManager @Inject constructor(@ApplicationContext private val context
     }
   }
 
-  /** Returns true iff the device has internet connectivity, false otherwise. */
+  /** Returns true if the device has internet connectivity, false otherwise. */
   @RequiresPermission("android.permission.ACCESS_NETWORK_STATE")
   fun isNetworkConnected(): Boolean {
-    val cm = context.getSystemService(Context.CONNECTIVITY_SERVICE) as ConnectivityManager
-    val networkInfo = cm.activeNetworkInfo
-    return networkInfo?.isConnected ?: false
+    val connectivityManager =
+      context.getSystemService(Context.CONNECTIVITY_SERVICE) as ConnectivityManager
+    val activeNetwork = connectivityManager.activeNetwork
```

Comment:
Can `activeNetwork` be null? If yes, can we add null check for safety?

Reply:
as the next LoC accepts null and eventually it will return false, so I think we don't need extra null check.

Reply:
SG